src/js/ol3-map-widget.js

```diff
@@ -916,6 +916,9 @@
 
                 marker_coordinates = ol.extent.getCenter(feature.getGeometry().getExtent());
                 marker_position = this.map.getPixelFromCoordinate(marker_coordinates);
+                var view = this.map.getView();
```

Comment:
Hi @fisuda.

Thank you very much for your contribution 😄. It LGTM although It'd be great if you can add a comment in this line providing the https://github.com/openlayers/openlayers/issues/4713 link as well as a new entry in the changelog.

Reply:
Hi @aarranz,

I have just added a comment and updated changelog.md. Please have a look.

Thank you.